models/prices/prices_usd_latest_old.sql

```diff
@@ -0,0 +1,23 @@
+{{ config(
+        schema='prices',
+        alias = alias('usd_latest_old'),
+        tags= ['static', 'dunesql'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['blockchain','contract_address','decimals','symbol']
```

Comment:
@jkylling by omitting `price` and `minute` this should update with the latest minute every time and the table should have one row per combination.

Reply:
just `blockchain` and `contract_address` should be enough here, as we enforce unique on `contract_address` for each blockchain-specific spell, then when we go cross-chain, `blockchain` will help differentiate

Reply:
for native tokens these can be null which would cause issues in that case?

Reply:
fair, it doesn't hurt to keep